packages/fluent/scss/stepper/_layout.scss

```diff
@@ -0,0 +1,291 @@
+            @include border-radius( calc(($kendo-stepper-indicator-height + 2 * $kendo-stepper-indicator-border-width) / 2) );
+            padding-inline: var( --kendo-stepper-label-padding-x, #{$kendo-stepper-label-padding-x} );
+            padding-block: var( --kendo-stepper-label-padding-y, #{$kendo-stepper-label-padding-y} );
+            border-width: 0;
+        }
+        .k-step-label .k-step-text {
+            max-width: calc(10em - (#{$kendo-icon-size} + #{$kendo-icon-spacing}));
+            flex-grow: 1;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+            overflow: hidden;
+        }
+        .k-step-label .k-icon {
+            margin-inline-start: $kendo-icon-spacing;
+        }
+
+        .k-step-current .k-step-label {
+            font-weight: bold;
+        }
+
+
+        // Optional label
+        .k-step-label-optional {
+            flex-basis: 100%;
+            font-size: var( --kendo-stepper-optional-label-font-size, #{$kendo-stepper-optional-label-font-size} );
+            font-style: var( --kendo-stepper-optional-label-font-style, #{$kendo-stepper-optional-label-font-style} );
+        }
+
+
+        .k-step-disabled,
+        .k-step.k-disabled {
+            opacity: 1;
+            pointer-events: none;
+
+            .k-step-link { cursor: default; }
+
+            .k-step-label-optional {
+                color: inherit;
+            }
+        }
+
+
+        // Progressbar
+        .k-progressbar {
+            pointer-events: none;
+            z-index: 0;
+            overflow: visible;
+        }
+
+        .k-progressbar-horizontal {
+            grid-row: 1 / -1;
+        }
+
+        .k-progressbar-vertical {
+            position: absolute;
+        }
+    }
+
+
+    // Horizontal
+    .k-step-list-horizontal {
+        flex-direction: row;
+        grid-row: 1;
+
+        .k-step {
+            flex: 1 0 auto;
+            text-align: center;
+        }
+
+        .k-step-link {
+            margin: auto;
+            max-width: 10em;
+            flex-direction: column;
+        }
+
+        // Label
+        .k-step-label {
+            text-align: center;
+        }
+        .k-step-indicator + .k-step-label {
+            margin-top: if($kendo-stepper-indicator-focus-offset > 0, calc(8px - #{$kendo-stepper-indicator-focus-offset}), 8px);
+        }
+
+        // Progressbar
+        & ~ .k-progressbar {
+            width: 100%;
+            height: var( --kendo-stepper-progressbar-size, #{$kendo-stepper-progressbar-size} );
+            top: calc(((#{$kendo-stepper-indicator-height} + 2 * #{$kendo-stepper-indicator-focus-size}) / 2) + #{$kendo-stepper-indicator-focus-size} / 2);
+        }
+    }
+
+    // Vertical
+    .k-step-list-vertical {
+        flex-direction: column;
+
+        .k-step {
+            min-height: calc((#{$kendo-stepper-indicator-width} + 2 * #{$kendo-stepper-indicator-border-width} + 2 * #{$kendo-stepper-indicator-focus-size}) + 20px);
```

Comment:
we should use CSS vars here, since we are using one for the kendo-stepper-indicator-width above. This is valid for all occurrences of the vars